specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/2016-12-01/backupManagement.json

```diff
@@ -1107,101 +1104,6 @@
         "deprecated": false
       }
     },
-    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/backupJobs": {
```

Comment:
Buggy on which side, the swagger, REST endpoint (or both)?  The removal will introduce breaking changes for clients using this API version.  Usually the pattern here is to fix the APIs in a new version and deprecate the existing ones so that clients don't break and can migrate to the new ones.